### ISSUE TITLE
AssertionError: rimraf: missing path

### ISSUE THREAD

**Issue by andypearson** [nib-health-funds/gulp-rev-delete-original#6]
I'm seeing the following error when using `gulp-rev-delete-original`:

```
assert.js:90
  throw new assert.AssertionError({
  ^
AssertionError: rimraf: missing path
    at rimraf (/project/node_modules/gulp-rev-delete-original/node_modules/rimraf/rimraf.js:49:3)
    at del (/project/node_modules/gulp-rev-delete-original/index.js:21:7)
    at DestroyableTransform._transform (/project/node_modules/gulp-rev-delete-original/index.js:61:14)
    at DestroyableTransform.Transform._read (/project/node_modules/gulp-rev-delete-original/node_modules/through2/node_modules/readable-stream/lib/_stream_transform.js:184:10)
    at DestroyableTransform.Transform._write (/project/node_modules/gulp-rev-delete-original/node_modules/through2/node_modules/readable-stream/lib/_stream_transform.js:172:12)
    at doWrite (/project/node_modules/gulp-rev-delete-original/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:237:10)
    at writeOrBuffer (/project/node_modules/gulp-rev-delete-original/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:227:5)
    at DestroyableTransform.Writable.write (/project/node_modules/gulp-rev-delete-original/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:194:11)
    at DestroyableTransform.ondata (stream.js:31:26)
    at emitOne (events.js:77:13)
```

I'm running a `gulp dist` task from a `gulpfile.js` that looks like:

```
var
  gulp = require('gulp'),
  rev = require('gulp-rev'),
  revdel = require('gulp-rev-delete-original');

gulp.task('dist', function() {
  return gulp.src('assets/**/*')
    .pipe(rev())
    .pipe(gulp.dest('assets/'))
    .pipe(revdel());
});
```

The contents of the `assets` directory is:

```
assets/
└── javascripts
    └── application.js
```

I'm using `0.2.1` from https://www.npmjs.com/package/gulp-rev-delete-original

**Comment by jameslnewell** [comment]
@andypearson Thanks for reporting! 

Can you please try `0.2.2` which I just published and contains #3, a pull-request that avoids `rimraf`ing `undefined` (a file which hasn't been revisioned by `gulp-rev` and therefore has no `revOrigPath`). 

If that's not the problem, can you please submit a PR to fix it. We're currently using gulp only in our legacy apps and we don't have time to take a look at it if it isn't affecting a project right now.

**Comment by andypearson** [comment]
@jameslnewell it's working great now! Cheers for the quick fix :)
